code/game/objects/items/weapons/pneumaticCannon.dm

```diff
@@ -26,6 +26,7 @@
 	var/fire_mode = PCANNON_FIREALL
 	var/automatic = FALSE
 	var/clumsyCheck = TRUE
+	var/trigger_guard = TRIGGER_GUARD_NORMAL
```

Comment:
https://github.com/tgstation/tgstation/blob/6df66f730f42c8ecea35427839a645d17fe17dc7/code/modules/mob/living/carbon/human/human_helpers.dm#L146

https://github.com/tgstation/tgstation/blob/11d96d9550d05e784f84231953a2abd7c1c9166c/code/modules/mob/living/living.dm#L796

It is used.